node/router/src/helpers/resolver.rs

```diff
@@ -13,15 +13,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use parking_lot::RwLock;
 use std::{collections::HashMap, net::SocketAddr};
 
 #[derive(Debug)]
```

Comment:
It would be good to add a doc comment for the resolver struct. (I thought I did that at some point, but maybe that change never got merged)